src/authentication/middleware/store_access_token_in_auth_service.js

```diff
@@ -5,8 +5,9 @@ export default (req, res, next) => {
   const authorizationHeaderValue = getRequestHeader('Authorization')
 
   // TODO: extract the token from the Authorization header value
-  const token = ''
-
-  setAccessToken(token)
-  next()
+  if (authorizationHeaderValue) {
+    const token = authorizationHeaderValue.split('')[1]
```

Comment:
This will split the whole string into an array of individual characters (see screenshot below). I think what you want to do here is split the string with a space.
```suggestion
    const token = authorizationHeaderValue.split(' ')[1]
```
![image](https://user-images.githubusercontent.com/12059563/113632901-5a2f7680-9642-11eb-8226-e62df9fc5e82.png)